### DISCUSSION TITLE
Search bar doesn't use colors from react navigation theme

### DISCUSSION THREAD

**Discussion by leatfingies** [software-mansion/react-native-screens#952] [Q&A]
It seems that search bar's appearance only gets affected by the OS theme, not the colors provided by react navigation. It'd also be great if there was a way to change the text color in the header options.

**Comment by tboba** [comment]
Hi @leatfingies, sorry for the late response 🙏 
Are you using `tintColor/barTintColor`, `headerIconColor` or `textColor/hintTextColor` to change the appearance of the search bar? If so, are you talking about the changes on iOS or Android? I know that React Navigation changes the naming of the parameters in options, but they should be similar 😄
